### ISSUE TITLE
why you return -torch.mean(loss)?

### ISSUE THREAD

**Issue by gengkai258** [anilbatra2185/road_connectivity#24]
understand! sorry!

**Comment by gengkai258** [comment]
understand!

**Comment by SunJiaHao33** [comment]
> understand! sorry!

Can you tell me why he return -torch.mean(loss)?
